packages/components/pie-toggle-switch/test/component/pie-toggle-switch.spec.ts

```diff
@@ -182,7 +182,7 @@ test.describe('Component: `Pie toggle switch`', () => {
 
     test.describe('Props: `LabelProps`', () => {
         test.describe('when a position is passed as `leading`', () => {
-            test('should render a leading label with class `c-toggleSwitch--leading`', async ({ mount }) => {
+            test('should render a leading label', async ({ mount }) => {
```

Comment:
To me it looks like these tests check for the label but not that it's in the correct position?

Reply:
Initially, the label element had a class name that indicated its position, allowing us to adjust its margin either to the right or left. but I updated the implementation to use the "gap" property instead. thus, the class name + test case isn't feasible anymore but we still ensured that the label container is defined if you pass the prop. 

Besides, I believe this type of test should be done in visual tests rather than component tests.

Reply:
If the DOM order changes based on whether it's leading or trailing, then I think it would be best done by inspecting the DOM as part of a unit/component test. If it's purely a styling change then I can see the argument for using a visual regression test (although those are less reliable and are slower and more costly to run).

Reply:
i added the unit tests as requested :)